drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -632,6 +632,23 @@ void RasterizerStorageGLES3::texture_allocate(RID p_texture, int p_width, int p_
 		p_flags &= ~VS::TEXTURE_FLAG_MIPMAPS; // no mipies for video
 	}
 
+#ifndef GLES_OVER_GL
+	switch (p_format) {
+		case Image::Format::FORMAT_RF:
+		case Image::Format::FORMAT_RGF:
+		case Image::Format::FORMAT_RGBF:
+		case Image::Format::FORMAT_RGBAF:
+		case Image::Format::FORMAT_RH:
+		case Image::Format::FORMAT_RGH:
+		case Image::Format::FORMAT_RGBH:
+		case Image::Format::FORMAT_RGBAH: {
+			// texture filtering not supported with float format on mobile (issue #24295)
+			p_flags &= ~VS::TEXTURE_FLAG_FILTER;
```

Comment:
Actually those formats are filterable if the extension `OES_texture_float_linear` is supported, as per "Issue (1)" in https://www.khronos.org/registry/OpenGL/extensions/OES/OES_texture_float_linear.txt
Could you add a check for this extension, and only disable filtering if not supported?

Reply:
Thanks for the review! It makes perfect sense, I've just updated with the change from your comments.